types/file-exists/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
Is it really necessary when `node` types are referenced?

Reply:
I think so; when I leave it outside of the array, TypeScript gives me an error `Cannot find name 'URL'`. Am I misunderstanding how to configure this file?

Reply:
Then it's ok.